src/audio/codec_adapter/codec/waves.c

```diff
@@ -0,0 +1,756 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2020 Waves Audio Ltd. All rights reserved.
+//
+// Author: Oleksandr Strelchenko <oleksandr.strelchenko@waves.com>
+//
+#include "sof/audio/codec_adapter/codec/generic.h"
+#include "sof/audio/codec_adapter/codec/waves.h"
+
```

Comment:
@stolx use ``<>`` instead of ``""`` for sof header inclusion.

Reply:
ok, changed to <>
could you please explain why?

Reply:
@stolx Coding style. Using "" for includes that aren't at a relative path is not accepted in the kernel coding style. You should use "" and <> accordingly (<> whenever that is usable).

Reply:
thanks for the explanation, will look closer at the kernel coding style

Reply:
Marking as resolved.